From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] Disable `Temporal` support

V8's implementation of `Temporal` is backed by `temporal_rs`, which would
require Zephyr projects to pull in the massive `temporal_capi` crate since
we can't statically link Rust code built with a potentially different compiler.
Since usage of `Temporal` is limited, the easiest fix is to just... not support
it at all.

---

diff --git a/.gn b/.gn
index 858a605..74b59b3 100644
--- a/.gn
+++ b/.gn
@@ -86,7 +86,7 @@ default_args = {
 
   enable_rust = true
   enable_rust_cxx = true
-  v8_enable_temporal_support = true
+  v8_enable_temporal_support = false
 
   # prevent hashdos
   v8_use_default_hasher_secret = false
diff --git a/src/lib.rs b/src/lib.rs
index f78efdc..3487937 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -27,7 +27,6 @@
 
 #[macro_use]
 extern crate bitflags;
-extern crate temporal_capi;
 
 mod array_buffer;
 mod array_buffer_view;
